doc/bt/common.bt

```diff
@@ -277,3 +277,13 @@ byte checkCompression()
 {
     return (ReadUShort() == 0xFA8D);
 }
+
+typedef struct
+{
+    short size;
+    short s1;
+
+    if(size > 0)
+        byte properties[size-2]; // todo, short:val list
```

Comment:
This structure can use something similar to the `ZC_OBJECT_PROPERTY.bt` template.